GeneralStateTests/stEIP1559/typeTwoBerlin.json

```diff
@@ -0,0 +1,86 @@
+{
+    "typeTwoBerlin" : {
+        "_info" : {
+            "comment" : "Ori Pomerantz qbzzt1@gmail.com",
+            "filling-rpc-server" : "evm version 1.10.4-unstable-950d99c2-20210602",
+            "filling-tool-version" : "retesteth-0.1.1-eip1559+commit.4d186352.Linux.g++",
+            "lllcversion" : "Version: 0.5.14-develop.2021.6.6+commit.401d5358.Linux.g++",
+            "source" : "src/GeneralStateTestsFiller/stEIP1559/typeTwoBerlinFiller.yml",
+            "sourceHash" : "6866687d172a5fc75bbf74790c08e6b83ac9d6a5bb8c34da51b9debb27bec475",
+            "labels" : {
+                "0" : ":label declaredKeyWrite"
+            }
+        },
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6",
+            "currentGasLimit" : "0xff112233445566"
```

Comment:
refill this tests with latest retesteth. now we have explicit basefee in env section

Reply:
You mean implicit, right?

Reply:
No. Now the standard is to have it explicit. So even legacy tests must have it in env.

Reply:
OK. It now has the basefee.